packages/lib-classifier/src/store/SubjectStore/SubjectStore.js

```diff
@@ -219,6 +228,34 @@ const SubjectStore = types
       }
     }
 
+    function * previous() {
+      const root = getRoot(self)
+      const workflow = tryReference(() => root.workflows.active)
+      const subject = tryReference(() => self.active)
+      const priority = subject?.priority
+      const subjectSetURL = `https://subject-set-search-api.zooniverse.org/subjects/${workflow.subjectSetId}.json`
+      const query = `priority__lt=${priority}&_size=1`
```

Comment:
Maybe we can request more than one subject here, in order to prime the `resources` map and speed up Previous navigation.